torch/_export/logical_schema.py

```diff
@@ -269,6 +269,18 @@ class Graph:
     symint_values: List[SymIntValue]
 
 
+@dataclass(init=False)
+class Epilogue:
+    inputs: List[TensorArgument]
```

Comment:
For the example above, 

inputs should have [index_for_first_index_put, value_for_firs_index_put, index_for_second_index_put, value_for_second_index_put]